Packs/Gatewatcher/Integrations/Gcenterv102/Gcenterv102.yml

```diff
@@ -0,0 +1,445 @@
+category: Network Security
+commonfields:
+  id: Gcenterv102
+  version: -1
+configuration:
+- display: GCenter IP address
+  name: ip
+  type: 0
+  required: true
+- display: GCenter API token
+  name: token
+  type: 0
+  required: false
+- display: GCenter username
+  displaypassword: GCenter password
+  name: credentials
+  type: 9
+  required: false
+- display: Check the TLS certificate
+  name: check_cert
+  type: 8
+  required: false
+description: This is the Gatewatcher integration for getting started.
```

Comment:
I suggest you change the description so it explains the purpose of this integration and the things it enables to its users.

Reply:
done